packages/portal/src/components/download/DownloadButton.vue

```diff
@@ -58,6 +58,10 @@
           return '_blank';
         }
         return '_self';
+      },
+      canonicalPageUrl() {
```

Comment:
Is there a reason we can't use the `canonicalUrl` mixin for this?

Reply:
only that `canonicalUrlWithoutLocale` uses $route.fullPath, which includes params & anchor tags, which would potentially cause different URLs being logged.

Reply:
I have refactored the mixin to have a single method supporting with/without locale, and path/fullPath.